fastlane/lanes/localization.rb

```diff
@@ -260,15 +260,11 @@
   ].freeze
 
   lane :update_frozen_strings_for_translation do
-    # We need to `cd` to the parent directory because, unlike when calling fastlane actions, commands running directly from the `Fastfile`
-    # (like `FileUtils` calls here) run relative to the `./fastlane` folder, but the `*_DIR_PATH` we use are relative to the repo root.
-    # See: https://docs.fastlane.tools/advanced/fastlane/#directory-behavior
```

Comment:
Ah, nice! With absolute paths, we don't need to worry about where in the filesystem the commands run.